assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -377,31 +377,33 @@ che.infra.kubernetes.server_strategy=default-host
 # Used to generate domain for a server in a workspace in case property `che.infra.kubernetes.server_strategy` is set to `multi-host`
 che.infra.kubernetes.ingress.domain=
 
+# DEPRECATED - please do not change the value of this property otherwise the existing workspaces will loose data. Do not
+# set it on new installations.
+#
```

Comment:
Shouldn't this be documented? Most people setting properties are not necessarily reading `che.properties` and instead overriding these values with environment variables. The potential for data loss is a little concerning.

Reply:
https://github.com/eclipse/che-docs/pull/855/files#diff-6283098727a587e6b8dcf3bafc9fc2d4R10